creator-node/src/apiHelpers.ts

```diff
@@ -95,33 +101,33 @@ export const handleResponseWithHeartbeat = <
 }
 
 export const sendResponse = (
-  req: Request,
+  _req: Request,
```

Comment:
can we not prefix params with underscore

Reply:
Ya I have a [PR on the way](https://github.com/AudiusProject/audius-protocol/pull/4061) that's set use an underscore prefix to signify a variable that's unused similar to how other languages do it (I haven't marked it for review yet cause I don't want to overwhelm everyone)

Reply:
@jonaylor89 thanks for the update. i'll change it to something else